Resources/doc/index.rst

```diff
@@ -64,8 +64,8 @@ Usage
 
 .. caution::
 
-    If your application is based on Symfony 3, replace ``php app/console`` by
-    ``php bin/console`` before executing any of the console commands included
+    If your application is based on Symfony <2.5.*, replace ``php bin/console`` by
```

Comment:
@Zuhayer The directory structure of the Symfony Standard Edition was changed with Symfony 3.